### PR TITLE
yaf: 2.6.0

### PR DESCRIPTION
**Pull request by vszakats**
yaf: use https, modernizing

### CODE REVIEW

**Comment by mikemcquaid** [comment]
@vszakats Please add tests and use `brew audit --strict` before submitting any more PRs.

**Comment by vszakats** [comment]
Got it, trying to make `brew audit` work as we speak.

**Comment by vszakats** [comment]
Test added. Audit passes.

**Comment by vszakats** [comment]
Set back to sha1.

File: Library/Formula/yaf.rb [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-    system './configure', "--disable-dependency-tracking",
+    system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/yaf", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/yaf.rb]: Looks like we could use http://tools.netsa.cert.org/yaf/libyaf/yaf_pcap.html and https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/test/fixtures/test.pcap to do something a bit more vigorous?

**Comment by vszakats** [comment]
Better test it is. I hope.

**Pull request closed without merging**